README.md

```diff
@@ -7,7 +7,6 @@
 </div>
 
 <!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
```

Comment:
- [ ] Good job making the suggested changes suggested by the last reviewer. However, you missed one important section which is `Future_Features`. This section lets you state possible future features the project will need to make it better. Things like sections in the HTML, proper styling, external dependency files, and possibly integration of third party libraries. You're free to put any number but a minimum of three.

- [ ] Changes should be made after the `Author` section, and the part that shows the description should not be added from the image which is present on the template given

## Missing section:
![image](https://github.com/siriwardhanage/Hello-Microverse/assets/50990925/bca52771-1bd3-4f0e-bc3f-1a80b8c7c3c2)